CHANGES.md

```diff
@@ -7,6 +7,7 @@ Next Release (5.7.0)
 
 Features
 --------
+* [#1238](https://github.com/java-native-access/jna/pull/1238): Add macOS `aarch64` architecture to universal `darwin` target. Cherry pick [libffi/libffi#577](https://github.com/libffi/libffi/pull/577). - [@fkistner](https://github.com/fkistner).
```

Comment:
@fkistner this commit no longer exists in this PR, was that intentional?

Reply:
This PR also requires the bumped libffi from your PR, but for review purposes I decided to only add my changes.
Added the relevant commits to this branch and rebased my changes on top including the cherry-pick.